### PR TITLE
Fix py-repl auto-generate=true

### PR DESCRIPTION
**Pull request by antocuni**
What is not tested is broken :man_facepalming: 
Test&fix `<py-repl auto-generate=true>` which was broken by PR #884

### CODE REVIEW

**Comment by marimeireles** [comment]
I'm gonna extra test this one.
Thanks @antocuni!

**Comment by marimeireles** [comment]
Beaautiful ✨

**Comment by antocuni** [comment]
> Beaautiful

plase approve it, else I cannot merge :man_facepalming:

**Review by marimeireles — APPROVED** [review]
Gr8!

File: pyscriptjs/tests/integration/support.py [hunk 1]
```diff
@@ -275,6 +275,15 @@ def pyscript_run(self, snippet, *, extra_head="", wait_for_pyscript=True):
         if wait_for_pyscript:
             self.wait_for_pyscript()
 
+    def iter_locator(self, loc):
+        """
+        Helper method to iterate over all the elements which are matched by a
+        locator, siny playwright does not seem to support it natively.
```
[the remark this answers is not in the document]
Comment (antocuni) [reply pyscriptjs/tests/integration/support.py]: thanks, fixed

**Pull request merged**